crates/bevy_asset/src/io/embedded/embedded_watcher.rs

```diff
@@ -24,6 +24,7 @@ pub struct EmbeddedWatcher {
 }
 
 impl EmbeddedWatcher {
+    /// Creates a new `EmbeddedWatcher` that watches for changes to the embedded assets in the given `dir`.
```

Comment:
I don't really think you need this. The comment above the struct is better.

Reply:
Yeah this is lint-proofing.